bioformats2raw/windows-bat-classpath-wildcard.patch

```diff
@@ -1,10 +1,10 @@
+-set CLASSPATH=%APP_HOME%\lib\bioformats2raw-0.3.0-rc3.jar;%APP_HOME%\lib\formats-gpl-6.5.1.jar;%APP_HOME%\lib\picocli-4.2.0.jar;%APP_HOME%\lib\univocity-parsers-2.8.4.jar;%APP_HOME%\lib\jzarr-0.3.3-gs-SNAPSHOT.jar;%APP_HOME%\lib\opencv-3.4.2-1.jar;%APP_HOME%\lib\progressbar-0.9.0.jar;%APP_HOME%\lib\formats-bsd-6.5.1.jar;%APP_HOME%\lib\formats-api-6.5.1.jar;%APP_HOME%\lib\ome-xml-6.1.0.jar;%APP_HOME%\lib\metakit-5.3.2.jar;%APP_HOME%\lib\ome-poi-5.3.3.jar;%APP_HOME%\lib\ome-codecs-0.3.0.jar;%APP_HOME%\lib\ome-common-6.0.4.jar;%APP_HOME%\lib\ome-mdbtools-5.3.2.jar;%APP_HOME%\lib\cdm-core-5.3.3.jar;%APP_HOME%\lib\logback-classic-1.1.1.jar;%APP_HOME%\lib\jxrlib-all-0.2.4.jar;%APP_HOME%\lib\native-lib-loader-2.1.4.jar;%APP_HOME%\lib\udunits-5.3.3.jar;%APP_HOME%\lib\httpservices-5.3.3.jar;%APP_HOME%\lib\slf4j-api-1.7.28.jar;%APP_HOME%\lib\JWlz-1.4.0.jar;%APP_HOME%\lib\minio-5.0.2.jar;%APP_HOME%\lib\joda-time-2.10.3.jar;%APP_HOME%\lib\kryo-2.24.0.jar;%APP_HOME%\lib\jhdf5-14.12.6.jar;%APP_HOME%\lib\json-20090211.jar;%APP_HOME%\lib\jackson-databind-2.10.3.jar;%APP_HOME%\lib\jblosc-1.0.1.jar;%APP_HOME%\lib\jline-3.16.0.jar;%APP_HOME%\lib\guava-28.0-jre.jar;%APP_HOME%\lib\logback-core-1.1.1.jar;%APP_HOME%\lib\xalan-2.7.2.jar;%APP_HOME%\lib\serializer-2.7.2.jar;%APP_HOME%\lib\specification-6.1.0.jar;%APP_HOME%\lib\turbojpeg-6.5.1.jar;%APP_HOME%\lib\jgoodies-forms-1.7.2.jar;%APP_HOME%\lib\perf4j-0.9.16.jar;%APP_HOME%\lib\metadata-extractor-2.11.0.jar;%APP_HOME%\lib\xercesImpl-2.8.1.jar;%APP_HOME%\lib\httpmime-4.5.9.jar;%APP_HOME%\lib\google-http-client-xml-1.20.0.jar;%APP_HOME%\lib\google-http-client-1.20.0.jar;%APP_HOME%\lib\httpclient-4.5.9.jar;%APP_HOME%\lib\commons-logging-1.2.jar;%APP_HOME%\lib\minlog-1.2.jar;%APP_HOME%\lib\objenesis-2.1.jar;%APP_HOME%\lib\commons-lang-2.4.jar;%APP_HOME%\lib\commons-io-1.4.jar;%APP_HOME%\lib\jdom2-2.0.6.jar;%APP_HOME%\lib\jsr305-3.0.2.jar;%APP_HOME%\lib\protobuf-java-3.9.1.jar;%APP_HOME%\lib\jcommander-1.78.jar;%APP_HOME%\lib\httpcore-4.4.11.jar;%APP_HOME%\lib\re2j-1.3.jar;%APP_HOME%\lib\jackson-annotations-2.10.3.jar;%APP_HOME%\lib\jackson-core-2.10.3.jar;%APP_HOME%\lib\junit-4.12.jar;%APP_HOME%\lib\jna-4.2.2.jar;%APP_HOME%\lib\okhttp-3.7.0.jar;%APP_HOME%\lib\okio-1.12.0.jar;%APP_HOME%\lib\xml-apis-1.3.04.jar;%APP_HOME%\lib\ome-jai-0.1.0.jar;%APP_HOME%\lib\jgoodies-common-1.7.0.jar;%APP_HOME%\lib\xmpcore-5.1.3.jar;%APP_HOME%\lib\failureaccess-1.0.1.jar;%APP_HOME%\lib\listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar;%APP_HOME%\lib\checker-qual-2.8.1.jar;%APP_HOME%\lib\error_prone_annotations-2.3.2.jar;%APP_HOME%\lib\j2objc-annotations-1.3.jar;%APP_HOME%\lib\animal-sniffer-annotations-1.17.jar;%APP_HOME%\lib\hamcrest-core-1.3.jar;%APP_HOME%\lib\xpp3-1.1.4c.jar;%APP_HOME%\lib\commons-codec-1.11.jar
 +set CLASSPATH=%APP_HOME%\lib\*

```

Comment:
Not an issue of this PR, but just a thought: It'd almost be easier to do this via a python script rather than a patch file.

Reply:
Agreed, being able to use regex would greatly remove the maintenance of this patch files as dependencies are being bumped. Captured as https://github.com/ome/conda-bioformats2raw/issues/6